e2e/deployments_test.go

```diff
@@ -0,0 +1,376 @@
+package e2e
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+	"time"
+)
+
+const (
+	subscriptionID = "076cd026-379c-4383-8bec-8835382efe90"
+	tenantID       = "72f988bf-86f1-41af-91ab-2d7cd011db47"
+	clientID       = "d1464cac-2a02-4e77-a1e3-c6a9220e99b9"
```

Comment:
Kindly if you can use the same ones that already passed to the e2e test and not add static values here.

Reply:
simplified this test to use cluster and variables created in aks.sh